06_modules_lab/03.py

```diff
@@ -1,11 +1,40 @@
-""" Write a program that searches current working directory
-for files larger than 1MB. Every time you find such a file print
-its name to the user.
-
-- When the program finds a large file. It should ask the user
-  a message asking if she wants to delete it, and delete the
-  file if requested
-
-- Take threshold and path as command line arguments
-"""
-
+import os 
+import sys
+
+print "whould you like to search in this folder(select 1) or other folder(select 2)"
+select_folder = raw_input()
+
+if select_folder == "1": 
+    new_folder = sys.argv [1]
+else:
+    new_folder = '.'
+
+
+print "whould you lile to select min size?Y\N"
+select_size = raw_input()
+
+if select_size == "y":
+    min_size = sys.argv [2] 
+else:
+    min_size = 1000000
+
+
+file_size = os.path.getsize(new_folder)
+
+for root,dirs,files in os.walk(new_folder):
+    for name in files:
+        file = (root + "\\" + name)
+        if file_size > min_size:
+            print "%s/%s" % (root,name)
+            print "This file size is over 1MB, Would you like to delete this file? Y\N"   
+            user_call = raw_input() 
+            while True:
+                if user_call == "y":
+                    os.remove(file)
+                elif user_call == "n":
+                    break
+                else:
+                    print "error, your input is wrong, try again"
+                    user_call = raw_input()
+
+

```

Comment:
It is better to prompt like this ```raw_input('text')``` instead of the separate print statement.
If you used argparse ( https://docs.python.org/2/library/argparse.html ) to get the parameters then you can make both of them optional with default values and you don't need to have the additional questions as you have now.